modules/virt-configuring-storage-class-bootsource-update.adoc

```diff
@@ -87,3 +92,19 @@ $ oc patch storageclass <current_default_storage_class> -p '{"metadata": {"annot
 $ oc patch storageclass <new_storage_class> -p '{"metadata":{"annotations":{"storageclass.kubernetes.io/is-default-class":"true"}}}' <1>
 ----
 <1> Replace `<new_storage_class>` with the `storageClassName` value that you added to the `HyperConverged` CR.
```

Comment:
So with the new file about changing the storage class, we can remove from here everything from line 62 to 94 :)